instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo/config.go

```diff
@@ -74,3 +79,17 @@ func WithCommandAttributeDisabled(disabled bool) Option {
 		cfg.CommandAttributeDisabled = disabled
 	})
 }
+
+// CommandTransformer defines a function that transforms a MongoDB command attribute.
```

Comment:
Can we document that if a transformer returns an empty string then the attribute is not set?